app-support/Kconfig

```diff
@@ -0,0 +1 @@
+rsource "wifi/Kconfig"
```

Comment:
There are options like:
```
config GOLIOTH_SAMPLE_WIFI
	bool "WiFi utilities for samples"
	depends on NET_L2_WIFI_MGMT
	help
	  Enable utilities for easy WiFi setup, mainly for use inside samples/.
```

So either we should place such options inside `sample_support` directory, or alternatively rename Kconfig options, so they would not mention "sample", but rather "app support" in their name.

Reply:
While these options are obviously tuned for our samples I'd like to imply they can be used by our users generically. In there applications.

Lets rename the Kconfig options to mention "app_support"